app/src/main/res/layout/inactive_header_item.xml

```diff
@@ -1,46 +1,58 @@
 <?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
+
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/inactive_header_border"
     android:layout_width="match_parent"
-    android:layout_height="48dp"
+    android:layout_height="wrap_content"
     android:layout_marginStart="16dp"
     android:layout_marginTop="12dp"
     android:layout_marginEnd="16dp"
     android:background="@drawable/card_list_row_background"
-    android:clickable="false"
-    android:clipToPadding="false"
-    android:focusable="true"
-    android:foreground="?android:attr/selectableItemBackground"
-    android:paddingStart="16dp"
-    android:paddingEnd="16dp">
+    android:backgroundTint="@color/photonWhite"
```

Comment:
We probably don't want `photonWhite` but the same thing we use as the divider. Since we aren't adding elevation, we can't follow the collections design entirely.

cc: @topotropic

Reply:
Thx for taking a look at this ❤️ 
Could you be more specific though? I just tried `neutral_faded_normal_theme` but it seems too faded:
![image](https://user-images.githubusercontent.com/60002907/131983506-ec4b9cfc-c978-4b36-b611-31d92d6ddda9.png)
![image](https://user-images.githubusercontent.com/60002907/131983572-45dc24db-e130-4714-bbd2-3009a2dc2762.png)

Reply:
@codrut-topliceanu sorry I didn't respond to this sooner.

How about we use `photonLightGrey30`? This seems consistent with what we use today for the grid view tab borders: https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/res/layout/tab_tray_grid_item.xml#L31-L32